cmd/stores/create_test.go

```diff
@@ -0,0 +1,82 @@
+package stores
```

Comment:
These were a different PR and unintentionally committed?

Reply:
no

Reply:
Split up the PR into two PRs (this one and one for UT only)